src/components/icon-button/icon-button.styles.ts

```diff
@@ -7,6 +7,7 @@ export default (disabled: boolean): SxStyleProp => ({
 
   height: '32px',
   width: '32px',
+  position: 'relative',
```

Comment:
is it necessary?
I don't see why we need this

Reply:
It is necessary, because the line 31 sets the svg's position as "position: 'absolute'", without this a bug would occur when icon-button is used in popup modal

Reply:
> is it necessary? I don't see why we need this

It is necessary, because the line 31 sets the svg's position as "position: 'absolute'", without this a bug would occur when icon-button is used in popup modal